### PR TITLE
Update cba_settings_userconfig

### PR DESCRIPTION
**Pull request by JamesTheClarke**
Updates cover two issues:

1.) all features added to ACE3 in the past year
2.) issue report by Dachi https://github.com/CntoDev/cba-settings-lock/issues/4

### CODE REVIEW

File: cba_settings_userconfig/cba_settings.sqf [hunk 1]
```diff
@@ -150,14 +156,14 @@ force force ace_maptools_rotateModifierKey = 1;
...
 force force ace_medical_AIDamageThreshold = 1;
-force force ace_medical_allowDeadBodyMovement = true;
 force force ace_medical_allowLitterCreation = true;
 force force ace_medical_allowUnconsciousAnimationOnTreatment = true;
 force force ace_medical_amountOfReviveLives = -1;
 force force ace_medical_bleedingCoefficient = 1;
 force force ace_medical_blood_enabledFor = 2;
 force force ace_medical_consumeItem_PAK = 0;
 force force ace_medical_consumeItem_SurgicalKit = 0;
+force ace_medical_convertItems = 0;
```
Comment (Gikerl) [inline cba_settings_userconfig/cba_settings.sqf]: Why is this disabled?
I am asking because I can't really find any source of what it is doing. But just from reading it, it seems to convert non-ACE medical supplies to ACE ones, which would be nice.

File: cba_settings_userconfig/cba_settings.sqf [hunk 2]
```diff
@@ -150,14 +156,14 @@ force force ace_maptools_rotateModifierKey = 1;
 // ACE Medical
 force force ace_medical_ai_enabledFor = 2;
 force force ace_medical_AIDamageThreshold = 1;
-force force ace_medical_allowDeadBodyMovement = true;
```
Comment (Gikerl) [inline cba_settings_userconfig/cba_settings.sqf]: Any reason why this was removed? What's the default value?

File: cba_settings_userconfig/$PREFIX$ [hunk 3]
```diff
@@ -0,0 +1 @@
+cba_settings_userconfig
```
Comment (Gikerl) [inline cba_settings_userconfig/$PREFIX$]: Why is this needed?

File: cba_settings_userconfig/cba_settings.sqf (hunk 2 above)
Comment (JamesTheClarke) [reply cba_settings_userconfig/cba_settings.sqf]: As far as I can tell this is no longer a valid configuration. I've loaded into a mission and exported all the current ACE3 settings after importing the server settings on top of them. That line no longer was shown in the export which leads me to believe its a remnant in our current server file that no longer does anything.

File: cba_settings_userconfig/cba_settings.sqf (hunk 1 above)
Comment (JamesTheClarke) [reply cba_settings_userconfig/cba_settings.sqf]: It's a fairly new feature of ACE3. I don't know how well it works nor have we tested it. That's why I left it on the default setting which is "off".

File: cba_settings_userconfig/$PREFIX$ (hunk 3 above)
Comment (JamesTheClarke) [reply cba_settings_userconfig/$PREFIX$]: I think this may be an unintended copy-paste error on my part when I was probably searching for something.
